docs/components/FileInputView.jsx

```diff
@@ -65,13 +102,21 @@ export default function FileInputView() {
             name: "label",
             type: "String",
             description: "Sets the label of the file input",
+            optional: true,
           },
           {
             name: "store",
             type: "Function",
             description: "Function to be called when the user has seleceted a file to be uploaded",
             defaultValue: "False",
           },
+          {
+            name: "displayLabels",
+            type: "Boolean",
+            description: "Display text labels inside of the component",
+            defaultValue: "true",
+            optional: true,
+          },
```

Comment:
curious, what was the reasoning behind having this prop as well, rather than relying on `label` existing or not?

Reply:
We thought it would be too confusing to have `label=""` cause many side-effects inside of the component. It hides multiple HTML elements, and seemed too magical.